src/components/ui/areaCard.tsx

```diff
@@ -21,16 +35,89 @@ const AreaCard = React.forwardRef<HTMLDivElement, AreaProps>((area, ref) => {
         <Button
           variant="outline"
           size="icon"
-          className="h-4 w-4 opacity-50 shrink-0 rounded-full hover:bg-accent hover:text-accent-foreground"
+          className="h-4 w-4 shrink-0 rounded-full opacity-50 hover:bg-accent hover:text-accent-foreground"
           onClick={area.toggleVisibility}
         >
           {area.visible ? <Icons.show /> : <Icons.hide />}
         </Button>
         <SheetTrigger>
-          <div className="h-4 w-4 opacity-50 shrink-0 rounded-full hover:bg-accent hover:text-accent-foreground">
+          <div className="h-4 w-4 shrink-0 rounded-full opacity-50 hover:bg-accent hover:text-accent-foreground">
             <Icons.info />
           </div>
         </SheetTrigger>
+        <SheetContent className="lg:min-w-[40vw]">
+          <SheetHeader>
+            <SheetTitle className="text-center">
+              <div>Area Details: </div>
```

Comment:
Maybe it would make sense to have a generic Sheet content component with these details structure if it is reused